src/main/java/org/mastodon/mamut/launcher/MastodonLauncher.java

```diff
@@ -627,7 +627,13 @@ private void loadMastodonProject( final String projectPath )
 					try
 					{
 						final MamutProject project = new MamutProjectIO().load( file.getAbsolutePath() );
-						windowManager.getProjectManager().open( project, true );
+						boolean projectSuccessfullyOpened = windowManager.getProjectManager().open( project, true );
```

Comment:
I don't like this boolean return value. My first idea would be to throw a CancellationException instead and catch it in appropriate places. I'm open to discuss this.

I see two use-cases for opening a Mastodon project: 1. with UI & user interaction 2. headless in a script.
And the code less and less supports the second usecase... (I don't know a good solution yet)